src/main/scala/mesosphere/marathon/metrics/MetricsReporterConf.scala

```diff
@@ -16,5 +16,12 @@ trait MetricsReporterConf extends ScallopConf {
     descr = "URL to dogstatsd agent. e.g. udp://localhost:8125?prefix=marathon-test&tags=marathon&interval=10",
     noshort = true
   )
+
+  lazy val averagingWindow = opt[Long](
```

Comment:
Nit: I think we encode the unit in the option name, e.g. `averagingWindowSizeSeconds`. I might be wrong though.

Reply:
That's a good idea 👍